### DISCUSSION TITLE
Do you have plans to support playlists?

### DISCUSSION THREAD

**Discussion by dipam7** [jdepoix/youtube-transcript-api#264] [Ideas]
I have a use case where I want to get a list of all videos in a playlist and then get the transcript for each video. Do you have plans to support something like this? Do you know any other python library that can reliably give a list of videos in a playlist?

**Comment by anthony-mg** [comment]
I had the same use case. The Youtube API is pretty simple to use and has a specific [endpoint just for playlists](https://developers.google.com/youtube/v3/docs/playlistItems)
